task/paths.go

```diff
@@ -16,6 +16,10 @@ func videoFileName(playbackID string) string {
 	return path.Join(playbackID, "video")
 }
 
+func mp4FileName(playbackID string) string {
+	return path.Join(playbackID, "mp4")
```

Comment:
@victorges if I understood correctly, this and the hls pathName should change to be a folder. Currently the hls pathname is `index.m3u8`, would you change it to `hls`?

Reply:
We only need to change the other file path if we are going with a single `output_location` in the request which is both HLS and MP4. If we are creating a separate `output_location` and parsing a separate `output` in the end, it doesn't really matter for now, and we can avoid changing any existing stuff.

If you do make any changes to use an array of a single `output_location`, the output folder should not be `hls`. It should be just `playbackId`, and the proper paths will be created internally by catalyst (`index.m3u8` or `hls/index.m3u8` for HLS, `mp4/*` for mp4s)